README.md

```diff
@@ -1,58 +1,172 @@
+
+**Note:** Many features of Pelican come from plugins, which might be tricky to use because of a version mismatch or inadequate documentation. Netlify supports Python and Pipenv but you still need to install your dependencies manually.
+
+### High-level overview
+
+|                                 | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
 |:--------------------------------|:------:|:------:|:------:|:-------:|
 | Single binary                   | ![yes] | ![yes] | ![yes] | ![no]   |
 | Language                        | Rust   | Rust   | Go     | Python  |
-| Syntax highlighting             | ![yes] | ![yes] | ![yes] | ![yes]  |
-| Sass compilation                | ![yes] | ![yes] | ![yes] | ![yes]  |
-| Assets co-location              | ![yes] | ![yes] | ![yes] | ![yes]  |
-| Multilingual site               | ![ehh] | ![no]  | ![yes] | ![yes]  |
-| Image processing                | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Sane & powerful template engine | ![yes] | ![yes] | ![ehh] | ![yes]  |
-| Themes                          | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Shortcodes                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Internal links                  | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Link checker                    | ![yes] | ![no]  | ![no]  | ![yes]  |
-| Table of contents               | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Automatic header anchors        | ![yes] | ![no]  | ![yes] | ![yes]  |
+| Syntax highlighting             | [![yes]](https://www.getzola.org/documentation/content/syntax-highlighting/) | ![yes] | ![yes] | ![yes]  |
+| Sass compilation                | [![yes]](https://www.getzola.org/documentation/content/sass/) | ![yes] | ![yes] | ![yes]  |
+| Multilingual site               | [![ehh]](https://www.getzola.org/documentation/content/multilingual/) | ![no]  | ![yes] | ![yes]  |
+| Image processing                | [![yes]](https://www.getzola.org/documentation/content/image-processing/) | ![no]  | ![yes] | ![yes]  |
+| Link checker                    | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#check) | ![no]  | ![no]  | ![yes]  |
+| Search                          | [![ehh]](https://www.getzola.org/documentation/content/search/) | ![no]  | ![no]  | ![yes]  |
+| Data files                      | [![yes]](https://www.getzola.org/documentation/templates/overview/#load-data) | ![yes] | ![yes] | ![no]   |
+| LiveReload                      | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#serve) | ![no]  | ![yes] | ![yes]  |
+| Configurable error/warn levels  | ![no]  | ![no]  | ![yes] | ? |
```

Comment:
I moved the "warn/error on broken links" here in the high-level overview and made more clear it's about error/warning management not about links. Is it better?